tests/e2e/streams/kafka.yml

```diff
@@ -1,13 +1,13 @@
-version: "3"
+version: '3.7'
```

Comment:
Why `3.7` ?

Reply:
It was like this in redpanda docs, I thought specifying a stricter version is okay.